### PR TITLE
AG-6686 - Allow configuration of axis title spacing and set theme def…

### PR DESCRIPTION
**Pull request by manapeirov**
PR for https://ag-grid.atlassian.net/browse/AG-6686

### CODE REVIEW

**Review by markdurrant — APPROVED** [review]
All LGTM 👍

**Review by alantreadway — CHANGES_REQUESTED** [review]
Code changes LGTM, please update snapshots.

**Comment by manapeirov** [comment]
@alantreadway PTAL

File: packages/ag-charts-community/src/chart/__image_snapshots__/cartesian-chart-test-ts-cartesian-chart-series-highlighting-should-correctly-change-highlighting-state-and-reset-1-snap.png
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/__image_snapshots__/cartesian-chart-test-ts-cartesian-chart-series-highlighting-should-correctly-change-highlighting-state-and-reset-1-snap.png:1]: This doesn't look right to me?

File: packages/ag-charts-community/src/chart/series/__image_snapshots__/series-labels-test-ts-series-labels-create-for-area-column-combo-series-labels-it-should-render-to-canvas-as-expected-1-snap.png
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/series/__image_snapshots__/series-labels-test-ts-series-labels-create-for-area-column-combo-series-labels-it-should-render-to-canvas-as-expected-1-snap.png:1]: Ditto

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**